createrUserSkeletonMethods

```diff
@@ -0,0 +1,54 @@
+from django.shortcuts import render, redirect, get_object_or_404
+from django.http import HttpResponseForbidden, HttpResponse
+from django.contrib.auth.decorators import login_required, user_passes_test
+from .models import User
+from .forms import UserForm 
+from django.contrib import messages
+
+
+class UserFormView:
+    # Handle GET requests to display the form
+    @login_required
+    def get(self, request, user_id=None):
```

Comment:
The user id will be passed in through the URL using url args. I don't think there's an example on main yet but I'm using it on my branch and can show you. Also this class already exists in "views/user_form/views.py"